mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -3895,6 +3950,24 @@ resource "google_container_cluster" "with_node_config" {
 `, clusterName)
 }
 
+func testAccContainerCluster_withNodePoolDefaults(clusterName, enabled string) string {
```

Comment:
I believe this needs similar version tags to prevent it from being present in the GA provider as an unused function